trie/utils.go

```diff
@@ -111,19 +111,36 @@ func (t *tracer) deleteList() [][]byte {
 		return nil
 	}
 	var ret [][]byte
-	for key := range t.delete {
-		ret = append(ret, []byte(key))
+	for path := range t.delete {
+		ret = append(ret, []byte(path))
 	}
 	return ret
 }
 
+// prevList returns the tracked node blobs in list format.
+func (t *tracer) prevList() ([][]byte, [][]byte) {
+	// Tracer isn't used right now, remove this check later.
+	if t == nil {
+		return nil, nil
+	}
+	var (
+		paths [][]byte
+		blobs [][]byte
+	)
+	for path, blob := range t.origin {
+		paths = append(paths, []byte(path))
+		blobs = append(blobs, blob)
+	}
+	return paths, blobs
+}
+
 // getPrev returns the cached original value of the specified node.
-func (t *tracer) getPrev(key []byte) []byte {
+func (t *tracer) getPrev(path []byte) []byte {
 	// Don't panic on uninitialized tracer, it's possible in testing.
```

Comment:
// Tracer isn't used right now, remove this check later. -> Update it for correcting the status.